tests/splines/non_periodic_spline_builder.cpp

```diff
@@ -82,9 +82,8 @@ TEST(NonPeriodicSplineBuilderTest, Identity)
 #elif defined(BSPLINES_TYPE_NON_UNIFORM)
         DVectX constexpr npoints(ncells + 1);
         std::vector<CoordX> breaks(npoints);
-        double dx = (xN - x0) / ncells;
         for (int i(0); i < npoints; ++i) {
-            breaks[i] = CoordX(x0 + i * dx);
+            breaks[i] = CoordX(x0 + i * (xN - x0) / ncells);
```

Comment:
Why this change? I find it more readable before. This change also changes the order of the operations. Is this intentional?

Reply:
Yeah I wanted to use `dx`  here: https://github.com/CExA-project/ddc/pull/560/files#diff-566ae334c766db6d714d50e38d0c0057788a2a11a014c079f4f6a4307bfdd2f0R202

Reply:
Does that cause a conflict? Doesn't the `{}` at lines 79 and 90 avoid any conflict with that line?

Reply:
Don't agree that this is resolved. I saw your change [d99d4af](https://github.com/CExA-project/ddc/pull/560/commits/d99d4af8f43eba924a3d14ef74b0f846946ac40f) but don't understand why the later `dx` was renamed. I would still like an answer to my last question

Reply:
I did not see your message. Indeed the `{}`, avoid conflict, I did not noticed it